src/mjolnir/ingest_transit.cc

```diff
@@ -143,13 +129,28 @@ struct unique_transit_t {
 };
 
 // Read from GTFS feed, sort data into the tiles they belong to
-std::priority_queue<tileTransitInfo> select_transit_tiles(const boost::property_tree::ptree& pt) {
+std::priority_queue<tile_transit_info_t> select_transit_tiles(const boost::property_tree::ptree& pt) {
 
   auto path = pt.get<std::string>("mjolnir.transit_feeds_dir");
 
   std::set<GraphId> tiles;
-  const auto& tile_level = TileHierarchy::levels().back();
-  std::unordered_map<GraphId, tileTransitInfo> tile_map;
+  const auto& local_tiles = TileHierarchy::levels().back().tiles;
+  std::unordered_map<GraphId, tile_transit_info_t> tile_map;
+
+  // adds a tile_info for the tile_id to tile_map if there is none yet, and returns the tile_info
+  // object
+  auto get_tile_info = [&tile_map, &local_tiles](const gtfs::Stop& stop) -> tile_transit_info_t& {
+    tile_transit_info_t tile_info;
+    auto graphid = GraphId(local_tiles.TileId(stop.stop_lat, stop.stop_lon),
+                           TileHierarchy::GetTransitLevel().level, 0);
+
+    auto tile_inserted = tile_map.insert({graphid, tile_info});
+    if (tile_inserted.second) {
+      tile_inserted.first->second.graphid = graphid;
+    }
+
+    return tile_inserted.first->second;
```

Comment:
nit: slightly verbose, the `::insert` method is made to do this work for you already
```suggestion
    return tile_map.insert({graphid, tile_transit_info_t{graphid}}).first->second;
```